executor/stale_txn_test.go

```diff
@@ -301,7 +415,7 @@ func (s *testStaleTxnSerialSuite) TestStalenessTransactionSchemaVer(c *C) {
 	tk.MustExec("create table t (id int primary key);")
 
 	schemaVer1 := tk.Se.GetSessionVars().GetInfoSchema().SchemaMetaVersion()
-	time.Sleep(time.Second)
+	time.Sleep(2 * time.Second)
```

Comment:
This may be unnecessary after #24820 is merged.

Reply:
Yes, it is.